src/xSTIR/pSTIR/STIR.py

```diff
@@ -1141,6 +1144,11 @@ def set_up(self, acq_templ, img_templ):
         '''
         assert_validity(acq_templ, AcquisitionData)
         assert_validity(img_templ, ImageData)
+
+        # temporary save the templates in the class
```

Comment:
grammar: "temporarily". However, it doesn't seem to be temporarily but permanently. So delete "temporary".

This has potentially impact on memory though. Do we really need to do this? They don't seem to be used anywhere.

Reply:
They are used in `get_linear_acquisition_model` and `get_background_term`